aws/validators.go

```diff
@@ -1386,6 +1386,33 @@ func validateCognitoUserPoolClientURL(v interface{}, k string) (ws []string, es
 	return
 }
 
+func validateCognitoResourceServerScopeDescription(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
This can be replaced with `validation.StringLenBetween(1, 256)`: https://godoc.org/github.com/hashicorp/terraform/helper/validation#StringLenBetween